expression/builtin_math.go

```diff
@@ -19,20 +19,17 @@ package expression
 
 import (
 	"fmt"
-	"hash/crc32"
-	"math"
-	"math/rand"
-	"strconv"
-	"strings"
-	"sync"
-	"time"
-
 	"github.com/pingcap/parser/mysql"
 	"github.com/pingcap/tidb/sessionctx"
 	"github.com/pingcap/tidb/types"
 	"github.com/pingcap/tidb/util/chunk"
 	"github.com/pingcap/tidb/util/mathutil"
 	"github.com/pingcap/tipb/go-tipb"
+	"hash/crc32"
+	"math"
+	"strconv"
+	"strings"
+	"sync"
```

Comment:
Please separate third-party libraries and official libraries.

Reply:
refactored.